openapi.yml

```diff
@@ -83,7 +83,7 @@ paths:
           required: true
           example: 123
```

Comment:
Looks like this may need to be made into a string.

Reply:
Yeah, I guess so.